src/main/java/com/mindway/server/v2/domain/book/presentation/BookController.java

```diff
@@ -25,26 +25,27 @@ public class BookController {
     private final GetBookListService getBookListService;
 
     @PostMapping
-    public ResponseEntity<Void> writeBookReport (@RequestBody @Valid BookWriteRequest bookWriteRequest) {
+    public ResponseEntity<Void> writeBookReport (
+            @RequestBody @Valid BookWriteRequest bookWriteRequest) {
         bookWriteService.execute(bookWriteRequest);
         return ResponseEntity.status(HttpStatus.CREATED).build();
     }
 
     @DeleteMapping("/{book_id}")
-    public ResponseEntity<Void> deleteBookReport (@PathVariable(value = "book_id") Long id) {
+    public ResponseEntity<Void> deleteBookReport (@PathVariable("book_id") Long id) {
         bookDeleteService.execute(id);
         return ResponseEntity.noContent().build();
     }
 
     @PatchMapping("/{book_id}")
     public ResponseEntity<Void> updateBookReport (
-            @PathVariable(value = "book_id") Long id, @RequestBody @Valid BookUpdateRequest bookUpdateRequest) {
+            @PathVariable("book_id") Long id, @RequestBody @Valid BookUpdateRequest bookUpdateRequest) {
```

Comment:
```suggestion
            @PathVariable("book_id") Long id, 
            @RequestBody @Valid BookUpdateRequest bookUpdateRequest
      ) {
```
이런 식으로 바꾸면 좋을 것 같아요 다른 것도 마찬가지로요...!

Reply:
[ce214e2](https://github.com/Team-MindWay/MindWay-v2-BackEnd/pull/138/commits/ce214e24de040e50d530bd54ded574389c67adcf)

수정했습니다